src/app/_interfaces/userProfile.interface.ts

```diff
@@ -0,0 +1,9 @@
+interface UserProfileInterface {
```

Comment:
userProfile.interface.ts should be user-profile.interface.ts => along with all other places where you use camelCase in names of files. There should be kebab-case